todoman/cli.py

```diff
@@ -466,6 +468,8 @@ def move(ctx, list, ids):
 @cli.command()
 @pass_ctx
 @click.argument('lists', nargs=-1, callback=_validate_lists_param)
+@click.option('--description',
```

Comment:
This flag was used [here](https://github.com/pimutils/todoman/pull/259/commits/09fcd2177af47abaabf3ab67973e0918ad73f38b#diff-d7b704f0d282081b24d6d106d9ea528fL656). By deleting that bit of code, this flag is now unused and ignored.

My initial comment on that related bit of code was that there were no tests for it (though the code itself seemed fine). Basically, that would be a test that uses `runner` with `'list', '--description'`.

You can either remove the flag (now unused), or undelete that code and add tests for it (I'm very much prefer the latter, of course. 🙂 ).